server/routes/api/v1/petTypesRouter.js

```diff
@@ -17,6 +17,9 @@ petTypesRouter.get("/", async (req, res) => {
 petTypesRouter.get("/:id", async (req, res) => {
   try {
     const petType = await PetType.findById(req.params.id)
+
+    petType.pets = await petType.findPets()
+
```

Comment:
We can remove the empty lines on 20 and 22 since this so clean it will still be easy to read 👍